src/components/layout/MaximizableView.js

```diff
@@ -26,7 +26,9 @@ function MaximizableView({ children }) {
   const maximizableElement = React.useRef(null);
   let isFullscreen;
   let setIsFullscreen;
-
+  const visibility = useSelector(
```

Comment:
I think this needs to fall back here to `DEFAULT_VISIBILITY` so that it's backwards compatible with settings where this key does not exist. Otherwise, it might just be `undefined`.

i.e. append an ` || DEFAULT_VISIBILITY;`, or something similar.